.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/go
+{
+	"name": "Authentik Terraform Provider",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "app",
+	"workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
+
+	"features": {
+		"ghcr.io/devcontainers/features/terraform:1": {}
+	},
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
```

Comment:
Might be useful to forward at least the authentik server 9000, for easier debugging and seeing what actually happens on the authentik instance.

Reply:
It already is actually! Ports added to this array here are just for VS Code to dynamically forward on startup to ports in your app container. In our case, the app container is the Terraform provider itself, so no forwarded ports are needed.
Ports forwarded in the Docker Compose are still honoured, so the Authentik server ports are forwarded and I actually added Postgres too for easier debugging there.
![image](https://github.com/goauthentik/terraform-provider-authentik/assets/1775711/2154b8d3-6daf-4d20-9369-ad6e5bc4f0d7)

Reply:
Ah, great!